tutorials/04_Syntehtic_Data_for_Machine_Learning.ipynb

```diff
@@ -101,14 +519,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "0.574182672682398\n"
+      "0.010323153621473069\n"
```

Comment:
why is the score so much worse on this dataset?

Reply:
Note that [this score](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.ElasticNet.html#sklearn.linear_model.ElasticNet.score) varies from 1 to negative infinity, and that training on the real data performs just as poorly, so I would guess it's simply a harder regression problem.